vcluster/_partials/manage/upgrade-version.mdx

```diff
@@ -144,26 +144,41 @@ Replace:
 Learn more about [Cluster API Provider for vCluster](https://github.com/loft-sh/cluster-api-provider-vcluster).
 
 <Flow>
+<Step>
+
+   [Install](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) the `clusterctl` CLI.
+
+</Step>
+<Step>
+
+   Install the vCluster provider.
+
+   ```bash
+   clusterctl init --infrastructure vcluster:v0.2.0
+   ```
+</Step>
 <Step>
 
    Export environment variables to be used by the cluster API provider to create an updated manifest. The manifest will be applied to your Kubernetes
    cluster, which will update your vCluster with the updated configuration options.
    
    ```bash
-    # Replace the vCluster variables with the values of the vCluster that you are updating
-    export CLUSTER_NAME=$VCLUSTER_NAME
-    export CLUSTER_NAMESPACE=$VCLUSTER_NAMESPACE
-    # Since the vcluster.yaml has changed, you'll need to export it to the updated vcluster.yaml
+    # Replace VCLUSTER_NAME and VCLUSTER_NAMESPACE with values of the vCluster that you are updating
+    export CLUSTER_NAME=<VCLUSTER_NAME>
+    export CLUSTER_NAMESPACE=<VCLUSTER_NAMESPACE>
+
+    # Since the vcluster.yaml has changed, you'll need to re-export it to the updated vcluster.yaml
     export VCLUSTER_YAML=$(awk '{printf "%s\\n", $0}' vcluster.yaml)
-    # Set what version of vCluster that you want to update
-    export CHART_VERSION=$VCLUSTER_VERSION
+    
+    # Set what version of vCluster that you want to update to
+    export CHART_VERSION=<VCLUSTER_CHART_VERSION>
     ```
     
     Replace:
 
-    - `$VCLUSTER_NAME` with the name of your running vCluster.
-    - `$VCLUSTER_NAMESPACE` with the namespace that is running the vCluster.
-    - `$VCLUSTER_VERSION` with the version of vCluster that you want to upgrade to.
+    - `<VCLUSTER_NAME>` with the name of your running vCluster.
+    - `<VCLUSTER_NAMESPACE>` with the namespace that is running the vCluster.
+    - `<VCLUSTER_CHART_VERSION>` with the version of vCluster that you want to upgrade to.
```

Comment:
Change this back to `<VCLUSTER_VERSION>` as it matches the variable across all the deployment methods.